src/courses/dto/courses.dto.ts

```diff
@@ -1,4 +1,6 @@
-import { IsString, IsUrl } from 'class-validator';
+import { Type } from 'class-transformer';
+import {IsArray, IsNumber, IsOptional, IsString, IsUrl } from 'class-validator';
```

Comment:
itt lemaradt sztem egy kis kód formázás, a { után célszerű tenni egy szóközt.
Amúgy kódformázásra a prettiert használjuk, be is van konfigolva a projektben elvileg by default, VSCode-hoz van plugin, érdemes azzal formázni mert az IDE-k között is konzisztens kódoformázást biztosít.

(van rá package.json-ben script is, npm run format, mert vscode extension aminek van billentyű kombinációja, mi még annyit szoktunk beletenni a projektben, hogy kommit előtt automatikusan formázzon hogy ne kerüljön be véletlenül sem formázatlan kód a repóba, de ez ide nem lett beépítve nyilván)